### PR TITLE
Made Changes

### PR DESCRIPTION
**Pull request by kautuk98**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All tests pass and I've made any necessary test changes.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

**Comment by Aaronius** [comment]
@kautuk98. Thanks for the contribution. Before we can accept the pull request, you'll need to sign the Adobe CLA: http://opensource.adobe.com/cla.html

I'll be adding a few inline questions to the pull request as well. Thanks!

File: karma.conf.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ governing permissions and limitations under the License.
 const puppeteer = require("puppeteer");
 const rollupConfig = require("./rollup.test.config");
 
-process.env.CHROME_BIN = puppeteer.executablePath();
+process.env.CHROME_BIN || 9876 = puppeteer.executablePath();
```
Comment (Aaronius) [inline karma.conf.js]: This is invalid code and is making tests fail. Can you describe what you were trying to accomplish here?

**Comment by jfkhoury** [comment]
Thank you for the contribution @kautuk98 but I agree with @Aaronius, we won't be merging this PR. But don't let that stop you from contributing in the future.

File: karma.conf.js (hunk 1 above)
Comment (kautuk98) [reply karma.conf.js]: Setting the ports altogether, could not understand why it was mentioned in the object.

**Comment by kautuk98** [comment]
Thanks to you, I learnt a thing or two.

File: src/utils/find.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ governing permissions and limitations under the License.
  * @returns {*}
  */
 export default (arr, predicate) => {
-  for (let i = 0; i < arr.length; i += 1) {
+  for (let i = 0; i < arr.length; i++) {      // i++ will execute faster than i+=1
```
[the remark this answers is not in the document]
Comment (kautuk98) [reply src/utils/find.js]: Got it! Thank you.

**Pull request closed without merging**